docker/README.md

```diff
@@ -113,4 +113,35 @@ docker run -p 8099:8099 pinot:release-0.1.0 StartBroker -zkAddress 172.17.0.2:21
 Example of bring up a local server:
 ```SHELL
 docker run -p 8098:8098 pinot:release-0.1.0 StartServer -zkAddress 172.17.0.2:2181
-```
\ No newline at end of file
+```
+
+## QuickStart
+
+
+### Use docker compose to bring up Pinot stack
+
+Please note that this quickstart is based on Kafka 2.0.
+
+Below is a script to use docker compose to bring up zookeeper/kafka/pinot-controller/pinot-broker/pinot-server
+```SHELL
+docker-compose -f docker-compose.yml up
+```
+
+### Create table and load data from Kafka.
+
+Below is the script to upload airlineStats schema
+```SHELL
+docker run --network=docker_default winedepot/pinot:kafka2 AddSchema -controllerHost pinot-controller -controllerPort 9000 -schemaFile sample_data/airlineStats_schema.json -exec
+```
+
+Below is the script to create airlineStats table
+```SHELL
+docker run --network=docker_default winedepot/pinot:kafka2 AddTable -filePath sample_data/docker/airlineStats_realtime_table_config.json -controllerHost pinot-controller -controllerPort 9000 -exec
+```
+
+Below is the script to ingest airplane stats data to Kafka
+```SHELL
```

Comment:
Can we make the docker image location as parameter?

Reply:
I feel this is a working example for demo/quickstart purposes. I don't expect users to directly put this script in production, but use a reference for set up.